### PR TITLE
Increase external network range

### PR DESCRIPTION
**Pull request by venkataanil**
We use the same CIDR(172.17.5.0/24) for floating ip in overcloud VMs.
But this won't be sufficient for scale testing. So use bigger
external network range.

Also set clean_debug to true by default, as most of the time
we will be deleting undercloud and redeploying.

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -81,15 +81,14 @@ dhcp_end: 192.168.24.105
 inspection_iprange: 192.168.24.110,192.168.24.250
 # external network params for adding external network to
 # undercloud to access overcloud resources
-external_gateway: 172.17.5.1/24
-external_network_broadcast: 172.17.5.255
+external_gateway: 172.18.0.1/16
 external_network_vlan_id: 300
-clean_debug: False
+clean_debug: True
```
Comment (smalleni) [inline group_vars/all.yml]: Why this change?

Comment (venkataanil) [reply group_vars/all.yml]: This will clean the HDD when we delete the overcloud.
This will avoid booting the overcloud node from the unwanted disk when it has multiple disks.

Comment (smalleni) [reply group_vars/all.yml]: Ack

Comment (venkataanil) [reply group_vars/all.yml]: Lets have this as a separate change as it is impacting node intropsection.
